CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 <!-- There is always Unreleased section on the top. Subsections (Added, Changed, Fixed, Removed) should be added as needed. -->
 
 ## Unreleased
+### Added
+- Fix: Package php-http/message-factory is abandoned, you should avoid using it. Use psr/http-factory instead.
```

Comment:
This is not a fix. It replaces underlying HTTP library interfaces, meaning the signatures of public methods will change. This is unfortunately a breaking change.

This also means there should be an UPGRADE document prepared showing a steps to upgrade (see root directory of the repository for previous upgrade document).

Reply:
But there is no changes needed. It will works without changes 🤷